subsys/bluetooth/mesh/dfu_srv.c

```diff
@@ -78,6 +78,8 @@ static void apply_rsp_sent(int err, void *cb_params)
 	struct bt_mesh_dfu_srv *srv = cb_params;
 
 	if (err) {
+		/* return phase back to give client one more chance. */
```

Comment:
We should perhaps remove the state storing at line 412 as well otherwise if response transmission failed and device rebooted, it will restore `BT_MESH_DFU_PHASE_APPLYING` instead of `BT_MESH_DFU_PHASE_VERIFY_OK`:
https://github.com/zephyrproject-rtos/zephyr/blob/4c2d27bcbec761f4bb78da27299eea6d305c3992/subsys/bluetooth/mesh/dfu_srv.c#L422-L423

`store_state` is called here anyway at line 100.